responsibleai/vision/components/rai_vision_insights/spec.yaml

```diff
@@ -0,0 +1,54 @@
+$schema: http://azureml/sdk-2-0/CommandComponent.json
+name: rai_vision_insights
+display_name: RAI Vision Insights
+version: 2
```

Comment:
for component in private / public preview, should we use semvar versioning and add preview at the end, for example 0.0.1.preview

Reply:
done!